bin/lib/get_cmd_args.sh

```diff
@@ -7,17 +7,32 @@
 # Globals:
 #   Sets the CONFIG variable
 #######################################
+function get_cmd_args::verify_config_file_exists() {
+  if [[ ! -f "${CONFIG}" ]]; then
+      echo "Config file named '${CONFIG}' was not found."
+      echo
+      echo "Create a file named civiform_config.sh or use the --config argument to specify a different file name."
```

Comment:
Should this be the value of CONFIG also?

If they set CONFIG to something other then civiform_config.sh, then this error should say that they need to create a file with that name, right?

Reply:
If you run `bin/doctor` or one of the other commands without specifying `--config` it defaults to looking for `civiform_config.sh`. So it sets the CONFIG to either the default name or what you passed in. The message will print the name of the file it was looking for on the first line. The second line talks about the default.

So you can create the default file name or point it to one you may already have.

That's how I came across this. I ran `bin/doctor` and it failed because I don't have a default file name anymore and wished I was told my options.

Reply:
Ah, I thought there might be confusion having it say
> Config file named 'not_default` was not found. Create a file name civiform_config.sh or use...

but if this is the standard way of phrasing this error message then it works for me.

Reply:
If you think of better wording I'm happy to change it.